helm_deploy/values-dev.yaml

```diff
@@ -14,5 +14,17 @@ generic-service:
     ENVIRONMENT_NAME: DEV
     AUDIT_ENABLED: "false"
 
+  # Pre-existing kubernetes secrets to load as environment variables in the deployment.
+  # namespace_secrets:
+  #   [name of kubernetes secret]:
+  #     [name of environment variable as seen by app]: [key of kubernetes secret to load]
+
+  namespace_secrets:
+    hmpps-contacts-ui:
+      SIGN_IN_CLIENT_ID: "SIGN_IN_CLIENT_ID"
+      SIGN_IN_CLIENT_SECRET: "SIGN_IN_CLIENT_SECRET"
+      SYSTEM_CLIENT_ID: "SYSTEM_CLIENT_ID"
+      SYSTEM_CLIENT_SECRET : "SYSTEM_CLIENT_SECRET"
+
```

Comment:
Sorry - I might have misled you here - these are not needed per env, as the base properties in values.yaml apply to all environments, and it reads the values from secrets in each env. So not needed in this file.